ios/Sources/AboutFeature/AboutLoadedScreen.swift

```diff
@@ -0,0 +1,192 @@
+import Component
+import ComposableArchitecture
+import Model
+import SwiftUI
+import Styleguide
+
+public struct AboutLoadedScreen: View {
```

Comment:
I think loaded is not screen. I think AboutLoadedView is better for this.

Reply:
I changed name to AboutLoadedView. d963558990a5e9b93158a74cc18ca2df489c5de9